### ISSUE TITLE
Produce fsli on deno-land

### ISSUE THREAD

**Issue by StoneCypher** [StoneCypher/fsl#920]

**Comment by StoneCypher** [comment]
Maybe impossible, maybe nevermind 😦 

Blocked by #921

**Comment by StoneCypher** [comment]
Closing as not planned: fsli (REPL, #8) was never built and deno.land/x is superseded by JSR; registry presence is tracked at #1503 and platform targets at v15.

Dispositioned in the #1849 legacy-milestone sweep.
